If you click the details, you can see this:

```
Run yarn run lint:prettier
Checking formatting...
[warn] src/panels/developer-tools/state/developer-tools-state.js
[warn] Code style issues found in the above file. Run Prettier to fix.
Error: Process completed with exit code 1.
```

This check is saying it doesn't like the way the code is formatted. The check is _very_ strict, not a single whitespace can be out of place. 

Typically if you have a real dev environment setup the `prettier` tool runs automatically as part of submit and formats your code in exactly the way it expects, so this would rarely hit anyone who was pushing from command line. If you try to make edits in the github UI I can see this happening. 

My guess is that it wants you to remove the blank line 64.

--- a/src/panels/developer-tools/state/developer-tools-state.js
+++ b/src/panels/developer-tools/state/developer-tools-state.js
@@ -61,7 +61,6 @@ class HaPanelDevState extends EventsMixin(LocalizeMixin(PolymerElement)) {
 
         .inputs {
           width: 100%;
-          
         }
 
         .info {

